index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width,initial-scale=1.0">
+
+  <!-- Favicon -->
+  <link rel="apple-touch-icon" sizes="180x180" href="/static/mhac_favicon/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/static/mhac_favicon/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/static/mhac_favicon/favicon-16x16.png">
+  <link rel="manifest" href="/static/mhac_favicon/site.webmanifest">
+  <link rel="mask-icon" href="/static/mhac_favicon/safari-pinned-tab.svg" color="#00aff0">
+
+  <meta name="msapplication-TileColor" content="#2784C3">
+  <meta name="theme-color" content="#2784C3">
+
+  <!-- Vue MultiSelect Default styles -->
+  <style src="vue-multiselect/dist/vue-multiselect.min.css"></style>
+
+  <title>MHAC | Midsouth Homeschool Athletics</title>
+
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-157989243-1"></script>
```

Comment:
I don't know if this has been updated on this branch but I have another branch that updates the Google Analytics

Reply:
I didn't update anything here, but it will end up with a problem, this file had to change location in Vue3